docs/content/users/install/phpstorm.md

```diff
@@ -4,19 +4,11 @@ If you work with the [PhpStorm](https://www.jetbrains.com/phpstorm/) IDE, you ca
 
 ## Before Starting
 
-Make sure to get at least one project going with `ddev start` before trying to set up the plugin, because the plugin assumes it has a project to work with.
-
-## Prerequisite
-
-Regardless of your setup, make PhpStorm use DDEV’s private `docker-compose` executable.
```

Comment:
I'm a little worried that I don't understand something about their new setup, but removing this is the best we can do at this point IMO. Thanks!